R/zchunk_LA119.solar_china.R

```diff
@@ -0,0 +1,79 @@
+#' module_gcam.china_LA119.Solar
+#'
+#' Compute capacity factors for wind by province.
+#'
+#' @param command API command to execute
+#' @param ... other optional parameters, depending on command
+#' @return Depends on \code{command}: either a vector of required inputs,
+#' a vector of output names, or (if \code{command} is "MAKE") all
+#' the generated outputs: \code{L119.CapFacScaler_PV_province}, \code{L119.CapFacScaler_CSP_province}. The corresponding file in the
+#' original data system was \code{LA114.Solar.R} (gcam-china level1).
+#' @details This chunk computes scalars by province to vary capacity factors for central station PV and CSP technologies by province.
```

Comment:
sth like: "computes solar capacity factor scalars for PV and CSP technologies by province, divided by the national average"

Reply:
OK